java/src/main/java/ai/rapids/cudf/Table.java

```diff
@@ -3631,7 +3631,12 @@ public static Table fromPackedTable(ByteBuffer metadata, DeviceMemoryBuffer data
       }
       result = new Table(columns);
     } catch (Throwable t) {
-      ColumnView.cleanupColumnViews(columnViewAddresses, columns);
+      // NOTE: Don't clean the columnViewAddresses as they were already cleaned by ColumnView
```

Comment:
This still can leak.  If we don't successfully complete converting all of the raw column view addresses into ColumnVectors then we leak all the views that weren't attempted for conversion.  Can probably fix this by leaving this as-is but instead of nulling the address _after trying to convert, we null it _before_ converting, that way we leave the cleanup of the one trying to be converted to `ColumnVecotr.fromViewWithContiguousAllocation` but all remaining ones can be cleaned up by `ColumnView.cleanupColumnViews` as before.